intbot/tests/test_integrations/github/issues.reopned.json

```diff
@@ -0,0 +1,255 @@
+{
```

Comment:
I used to do this, then I started to use vcr. And then I started to get rid of this stuff as much as possible and keep only the json parts I needed.

Reply:
yup, I mostly do the same. Most of the tests in this PR are actually using parts defined in the tests. This is there mostly for reference and those few tests that are parsing everything – by copy pasting entire webhook we also make sure we don't have any typos or some accidental change in structure.